### PR TITLE
Readme: update description to include Site Accelerator.

### PR DESCRIPTION
**Pull request by jeherve**
#### Changes proposed in this Pull Request:

Make changes to the plugin description on WordPress.org, based on feedback on p6TEKc-2mI-p2

#### Testing instructions:

1. Check the readme; it should match the changes requested at the link above.

#### Proposed changelog entry for your changes:
* None.

### CODE REVIEW

**Review by kraftbj — APPROVED** [review]
Still needs to be reviewed by @joanrho. Approving it as a code owner so it can be merged immediately if she approves.

**Review by joanrho — APPROVED** [review]
Code looks great. Thanks @jeherve!

**Pull request merged**
